notebooks/anisotropic_representations.ipynb

```diff
@@ -344,10 +344,10 @@
      "evalue": "No module named 'anisoap.representations.projection_coefficients_modified'",
      "output_type": "error",
      "traceback": [
-      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
-      "\u001b[0;31mModuleNotFoundError\u001b[0m                       Traceback (most recent call last)",
-      "Cell \u001b[0;32mIn [47], line 1\u001b[0m\n\u001b[0;32m----> 1\u001b[0m \u001b[38;5;28;01mfrom\u001b[39;00m \u001b[38;5;21;01manisoap\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mrepresentations\u001b[39;00m\u001b[38;5;21;01m.\u001b[39;00m\u001b[38;5;21;01mprojection_coefficients_modified\u001b[39;00m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;241m*\u001b[39m\n",
-      "\u001b[0;31mModuleNotFoundError\u001b[0m: No module named 'anisoap.representations.projection_coefficients_modified'"
+      "\u001B[0;31m---------------------------------------------------------------------------\u001B[0m",
```

Comment:
You may want to do `nbstripout` before committing so that you don't get small tracked changes like these.

Reply:
All prior notebooks have been uncommitted in this PR.